pandas/core/indexes/datetimes.py

```diff
@@ -3,7 +3,7 @@
 from datetime import (
     date,
     datetime,
-    time,
+    time as dt_time,
     timedelta,
     tzinfo,
 )
```

Comment:
can we `import datetime as dt` and then access all these from `dt.`?

Reply:
Sounds good! I'll make these changes and commit.